Frends.SMTP.SendEmail/Frends.Smtp.SendEmail/SendEmailTask.cs

```diff
@@ -9,143 +9,153 @@
-        /// </summary>
-        private static SmtpClient InitializeSmtpClient(Options settings)
-        {
-            var smtpClient = new SmtpClient
-            {
-                Port = settings.Port,
-                DeliveryMethod = SmtpDeliveryMethod.Network,
-                UseDefaultCredentials = settings.UseWindowsAuthentication,
-                EnableSsl = settings.UseSsl,
-                Host = settings.SMTPServer
-            };
-
-            if (!settings.UseWindowsAuthentication && !string.IsNullOrEmpty(settings.UserName))
-                smtpClient.Credentials = new NetworkCredential(settings.UserName, settings.Password);
-
-            return smtpClient;
-        }
-
-        /// <summary>
-        /// Initializes new MailMessage with given parameters. Uses default value 'true' for IsBodyHtml
-        /// </summary>
-        private static MailMessage InitializeMailMessage(Input input)
-        {
-            //split recipients, either by comma or semicolon
-            var separators = new[] { ',', ';' };
-
-            string[] recipients = input.To.Split(separators, StringSplitOptions.RemoveEmptyEntries);
-            string[] ccRecipients = input.Cc.Split(separators, StringSplitOptions.RemoveEmptyEntries);
-            string[] bccRecipients = input.Bcc.Split(separators, StringSplitOptions.RemoveEmptyEntries);
-
-            //Create mail object
-            var mail = new MailMessage()
-            {
-                From = new MailAddress(input.From, input.SenderName),
-                Subject = input.Subject,
-                Body = input.Message,
-                IsBodyHtml = input.IsMessageHtml
-            };
-            //Add recipients
-            foreach (var recipientAddress in recipients)
-            {
-                mail.To.Add(recipientAddress);
-            }
-            //Add CC recipients
-            foreach (var ccRecipient in ccRecipients)
-            {
-                mail.CC.Add(ccRecipient);
-            }
-            //Add BCC recipients
-            foreach (var bccRecipient in bccRecipients)
-            {
-                mail.Bcc.Add(bccRecipient);
-            }
-
-            //Set message encoding
-            Encoding encoding = Encoding.GetEncoding(input.MessageEncoding);
-
-            mail.BodyEncoding = encoding;
-            mail.SubjectEncoding = encoding;
-
-            return mail;
-        }
-
-        /// <summary>
-        /// Gets all actual file names of attachments matching given file path
-        /// </summary>
-        private static ICollection<string> GetAttachmentFiles(string filePath)
-        {
-            string folder = Path.GetDirectoryName(filePath);
-            string fileMask = Path.GetFileName(filePath) != "" ? Path.GetFileName(filePath) : "*";
-
-            string[] filePaths = Directory.GetFiles(folder, fileMask);
-
-            return filePaths;
-        }
-    }
-}
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
pragma IDE1006 used because there was an error related to the Pascal naming convention and only pragma could prevent these warnings

Reply:
Please enable the warning immediately after the problem place.